Doc/library/typing.rst

```diff
@@ -500,6 +504,8 @@ The module defines the following classes, functions and decorators:
 
     A generic version of :class:`collections.abc.Reversible`.
 
+   .. versionadded:: 3.5.2
```

Comment:
According to the devguide, it is 3, but don't worry about that too much. You mainly need to align the `..versionadded` with the preceding text so they are formatted as part of the same block.